.github/workflows/build.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         run: npm run build-storybook
 
       - name: Upload Artifact
-        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
```

Comment:
interesting - the hash here hasn't changed but the version has. Is this intentional?

Reply:
Is the `# v4` pinning to the major version of 4, while this will pin to the exact version?

Reply:
yup. It will also allow for semver bumps inside `renovate`